Formula/fdroidserver.rb

```diff
@@ -353,7 +353,7 @@ def install
                 "JPEG_ROOT = ('#{Formula["jpeg"].opt_prefix}/lib', '#{Formula["jpeg"].opt_prefix}/include')"
         s.gsub! "FREETYPE_ROOT = None",
                 "FREETYPE_ROOT = ('#{Formula["freetype"].opt_prefix}/lib', " \
-                                 "'#{Formula["freetype"].opt_prefix}/include')"
+                "'#{Formula["freetype"].opt_prefix}/include')"
```

Comment:
This is also not nice. Is it really too long to be one line?

Reply:
Will be fixed in https://github.com/Homebrew/homebrew-core/pull/80457